Geometry/TrackerGeometryBuilder/test/BuildFile.xml

```diff
@@ -2,6 +2,7 @@
 <use name="Geometry/TrackerNumberingBuilder"/>
 <use name="Geometry/CommonDetUnit"/>
 <use name="FWCore/Framework"/>
+<use name="cuda"/>
```

Comment:
This would be better under `<bin file="phase1PixelTopology_t.cu">` to avoid making all tests depend on `cuda`.